tools/example-showcase/src/main.rs

```diff
@@ -692,7 +692,14 @@ fn parse_examples() -> Vec<Example> {
                 path: val["path"].as_str().unwrap().to_string(),
                 name: metadata["name"].as_str().unwrap().to_string(),
                 description: metadata["description"].as_str().unwrap().to_string(),
-                category: metadata["category"].as_str().unwrap().to_string(),
+                category: metadata["category"]
+                    .as_str()
+                    .unwrap()
+                    .replace(['(', ')'], "")
+                    .to_lowercase()
+                    .split_whitespace()
+                    .collect::<Vec<_>>()
+                    .join("-"),
```

Comment:
maybe
```suggestion
                    .replace([' '], "-"),
```

Reply:
Yeah, guess that should also be fine or the current set of categories.